packages/react/src/patterns/blocks/LeadSpaceBlock/README.md

```diff
@@ -0,0 +1,143 @@
+      ensuring software is always ready for deployment.`;
+  
+      const mediaType = select(
+        'mediaType (optional)',
+        ['image', 'video', 'none'],
+        'image'
+      );
+  
+      const video = {
+        videoId: '0_uka1msg4',
+        showDescription: true,
+      };
+  
+      const image = {
+        heading: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+        image: {
+          sources: [
+            {
+              src: 'https://dummyimage.com/320x180/ee5396/161616&text=16:9',
+              breakpoint: 320,
+            },
+            {
+              src: 'https://dummyimage.com/400x225/ee5396/161616&text=16:9',
+              breakpoint: 400,
+            },
+            {
+              src: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+              breakpoint: 672,
+            },
+          ],
+          alt: 'Image alt text',
+          defaultSrc: 'https://dummyimage.com/672x378/ee5396/161616&text=16:9',
+        },
+      };
+  
+      const mediaData = mediaType === 'image' ? image : video;
+  
+      const linkListProps = {
+        heading: text('link list heading:', 'Featured products'),
+        items: object('link list items array', [
+          {
+            type: 'local',
+            copy: 'IBM Cloud Continuous Delivery',
+            cta: {
+              href: 'https://ibm.com',
+            },
+          },
+          {
+            type: 'local',
+            copy: 'UrbanCode',
+            cta: {
+              href: 'https://ibm.com',
+            },
+          },
+          {
+            type: 'local',
+            copy: 'View all products',
+            cta: {
+              href: 'https://ibm.com',
+            },
+          },
+        ]),
+      };
+
+  return (
+    <LeadSpaceBlock title={title} 
+                    heading={heading}
+                    copy={copy}
+                    mediaType={mediaType}
+                    mediaData={mediaData} 
+                    items={linkListProps}
+                />
+  );
+}
+
+ReactDOM.render(<App />, document.querySelector('#app'));
+```
+
+## Props
+
+| Name        | Required | Data Type | Description                       |
+| ---------   | -------- | --------- | ----------------------------------|
+| `title  `   | YES      | String    | heading of the content block.     |
+| `heading`   | YES      | String    | subheading of the content block.  |
+| `copy`      | YES      | String    | Simple content item.              |
+| `mediaType` | NO       | string    | Media Type [image, video or none] |
+| `mediaData` | NO       | Object    | Media Data                        |
+| `items`     | YES      | Object    | Link list items                   |
+| `button`    | NO       | Object    | Button props                       |
+
+### Stable selectors
+
+| Name                             | Description |
+| -------------------------------- | ----------- |
+| `dds--lead-space-block`          | Pattern     |
```

Comment:
let's be consistent with the stable selector naming pattern, let's go with this instead:

```
dds-leadspace-block`
```

Reply:
used the selector as mentioned above.

Reply:
@greeshmareddyg I'm not seeing a bunch of the changes you mentioned, did you push them up?

Reply:
@jeffchew could you check the changed files.